src/generator.ts

```diff
@@ -65,6 +65,12 @@ export class GenType extends fc.Arbitrary<any> {
     if (type.type === 'object') {
       return new GenObject(type.properties, type.additionalProperties);
     }
+    if (type.type === 'boolean') {
+      if (type.enum) {
+        return fc.oneof(...type.enum.map(fc.constant));
```

Comment:
can there be enum booleans? 🤔

Reply:
I guess. actually I think that we can have `enum: true, 2, 'xx' ` even but that is so silly that I did not want to deal with it.

Reply:
please lets not 😅